tests/foreman/cli/test_org.py

```diff
@@ -226,36 +207,17 @@ def test_bugzilla_1076568_2(self):
         @BZ: 1076568
 
         """
+        org = make_org()
 
-        new_obj = make_org()
-
-        # Can we find the new object?
```

Comment:
Isn't this a valid test? You just deleted the entity and now you make sure that you cannot find it?